ciscoise_connector.py

```diff
@@ -727,6 +735,92 @@ def _clear_policy(self, param):
         action_result.add_data(ret_data)
         return action_result.set_status(phantom.APP_SUCCESS, "Policy cleared")
 
+    def _list_policies(self, param):
+
+        ret_val = phantom.APP_SUCCESS
+
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        ret_data = None
+        endpoint = ERS_POLICIES
+
+        ret_val, ret_data = self._call_ers_api(endpoint, action_result)
+
+        if phantom.is_fail(ret_val):
+            return action_result.get_status()
+
+        total = ret_data["SearchResult"]["total"]
+        policies = ret_data["SearchResult"]["resources"]
+
+        for policy in policies:
+            endpoint = f"{ERS_POLICIES}/{policy['id']}"
+
+            ret_val, ret_data = self._call_ers_api(endpoint, action_result)
+            if phantom.is_fail(ret_val):
+                return action_result.get_status()
+            data = ret_data["ErsAncPolicy"]
+            data['actions'] = ', '.join(data['actions'])
+            action_result.add_data(data)
+
+        action_result.update_summary({"policies_found": total})
+
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def _delete_policy(self, param):
+
+        ret_val = phantom.APP_SUCCESS
+
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        ret_data = None
```

Comment:
remove this from here as well from all other places where it doesn't have any uses.